quick-setup.py

```
@@ -0,0 +1,369 @@
+- Installs dependencies with proper error handling
+- Downloads required NLTK data
+- Provides troubleshooting guidance
+
+Usage:
+    python quick-setup.py
+    python quick-setup.py --install-ollama
+    python quick-setup.py --help
+"""
+
+import sys
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)